api/src/test/resources/org/openmrs/api/db/hibernate/include/HibernatePatientDAOTest-patients.xml

```diff
@@ -68,7 +68,7 @@
         Voided patient with non-voided name
     -->
     <person person_id="60" gender="M" birthdate="1998-09-30 12:34:56.7" birthdate_estimated="0" dead="false" creator="1" date_created="2014-01-09 00:00:00.0" voided="false" uuid="7443D9DD-CA51-4BD9-9E68-7C43628B82DF"/>
-    <person_name person_name_id="60" preferred="true" person_id="60" prefix="Mr." given_name="Meriadoc" middle_name="" family_name="Brandybuck" family_name2="" creator="1" date_created="2014-01-09 00:00:00.0" voided="false" void_reason="" uuid="61ACD71F-989B-4B4A-B00E-0503B756099F"/>
+    <person_name person_name_id="60" preferred="true" person_id="60" prefix="Mr." given_name="Meriadoc" middle_name="" family_name="Brandybuck" family_name2="" creator="1" date_created="2014-01-09 00:00:00.0" voided="true" void_reason="" uuid="61ACD71F-989B-4B4A-B00E-0503B756099F"/>
```

Comment:
This was a construed scenario that should never arise based on the semantics of our API because a voided patient should never have a non-voided voided name, If an admin tinkers with data in the DB, they better know what they are doing.